federation/develop/develop.sh

```diff
@@ -0,0 +1,218 @@
+#!/usr/bin/env bash
+
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script will build the hyperkube image and push it to the repository
+# referred to by KUBE_REGISTRY. The image will be given a version tag with
+# the value from KUBE_VERSION.
+# e.g. run as:
+# KUBE_REGISTRY=localhost:5000/anushku \
+# KUBE_VERSION=1.3.0-dev ./build.sh build_image
+#
+# will build the Docker images with the specified repository name and the
```

Comment:
and actually deploy as well. ok, we cant mention everything here.
Fine with whatever you think is reasonable.